src/zookeeper.ts

```diff
@@ -142,13 +111,40 @@ class ZookeeperLocatorFatory extends EventEmitter {
         // ugly, but this whole shared event emitter business sort of is :(
         // also, how only the zookeeper locator implements this :(
         Object.keys(EventEmitter.prototype).forEach((fnName) =>
-            (<any>this.getLocatorForPath)[fnName] = this[fnName]
+            ((this as any).getLocatorForPath)[fnName] = this[fnName]
```

Comment:
not so easily, i wasn't looking to hard into changing the logic, just updating stuff, but i can dig in if you'd really like